CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 0.14.0
 
+- Adds new 'limiter' class for setting max page width across screen sizes.
+- Significantly refactors layout scale to be more consistent across different properties and to increment more predictibly.
```

Comment:
I'd appreciate mention of the specific numbers that were added and removed. I got stumped by `py48` no longer working, and had to dig into the commit history to figure out what happened.